### DISCUSSION TITLE
Welcome to faceit-rating-system-analysis Discussions!

### DISCUSSION THREAD

**Discussion by ollipasanen** [ollipasanen/faceit-rating-system-analysis#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by uzhuzhh** [comment]
Super cool stuff! Thanks for the analysis. I was looking to rate my team's success in our local league by using faceit rating. We play in a Cup hosted on faceit and are quite the underdog. I wanna see how many wins would be expected according to Faceit's win probability model. However, Faceit does not provide the win probability in the match room anymore if it is not a ranked match, it seems. In your analysis, you mentioned the win probability by elo difference ("1k Elo = 99.7% win prob"). Where did you get the number from? Do you happen to have the full chart? I am looking for more realistic matchups like 2100 avg elo vs 2500 avg elo and so on. 
Love your work and thanks in advance!

**Reply by ollipasanen** [reply]
Hey! First of all, I believe faceit has changed their rating system internally multiple times after I've published this repo, so it's most likely not up-to-date information anymore. I think they use some machine learning based ratings now, but I could be wrong.

The probability to win in the Elo system is defined by the difference in ratings, not their absolute value. If you have a 400 point difference, the odds are 10 : 1. The probability is the same even if you have 1000 rating against 1400 or 2000 against 2400. So, in your case 2100 VS 2500 will be around 91% chance of losing. The actual probability can be calculated with 1/(10 ^ (difference / 400) + 1) in the typical implementation. 

Here's one calculator that's available online: https://sandhoefner.github.io/chess.html
